### PR TITLE
DOCS-3301: Document Fix Credential Button and UI Function

### PR DESCRIPTION
**Pull request by jervin777**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md [hunk 1]
```diff
@@ -29,7 +29,21 @@ Go to **Data Protection > Cloud Sync Tasks** and click **Add**.
 
 ![DataProtectionCloudSyncAdd](/images/SCALE/DataProtectionCloudSyncAdd.png "Creating a Cloud Sync Task")
 
-Type a memorable task description in the **Description** field. Use the **Credential** dropdown list to select an existing cloud or create a new one with the **+ Add a backup credential** option. TrueNAS connects to the chosen cloud storage provider and shows the available storage locations. Select the option if data is transferring to (**PUSH**) or from (**PULL**) the cloud storage location (**Remote**). Select a **Transfer Mode**:
+Type a memorable task description in the **Description** field. Use the **Credential** dropdown list to select an existing cloud or create a new one with the **+ Add a backup credential** option. 
+
+{{< expand "What happens if my cloud sync credentials are invalid?" "v" >}}
+Once you chose a cloud credential from the dropdown list, TrueNAS automatically validates access to that cloud sync provider.  Invalid credentials will result in the following Alert:
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md:35]:
```suggestion
Once you choose a cloud credential from the dropdown list, TrueNAS automatically validates access to that cloud sync provider.  Invalid credentials results in the following Alert: 
```

File: content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md [hunk 2]
```diff
@@ -29,7 +29,21 @@ Go to **Data Protection > Cloud Sync Tasks** and click **Add**.
...
 
-Type a memorable task description in the **Description** field. Use the **Credential** dropdown list to select an existing cloud or create a new one with the **+ Add a backup credential** option. TrueNAS connects to the chosen cloud storage provider and shows the available storage locations. Select the option if data is transferring to (**PUSH**) or from (**PULL**) the cloud storage location (**Remote**). Select a **Transfer Mode**:
+Type a memorable task description in the **Description** field. Use the **Credential** dropdown list to select an existing cloud or create a new one with the **+ Add a backup credential** option. 
+
+{{< expand "What happens if my cloud sync credentials are invalid?" "v" >}}
+Once you chose a cloud credential from the dropdown list, TrueNAS automatically validates access to that cloud sync provider.  Invalid credentials will result in the following Alert: 
+
+![DataProtectionCloudSyncInvalidAlert](/images/SCALE/DataProtectionCloudSyncInvalidAlert.png "Invalid Credentials Alert")
+
+Click **FIX CREDENTIAL**.  You will be directed to the Cloud Credentials entry view.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md:39]:
```suggestion
Click **FIX CREDENTIAL**.  TrueNAS directs you to the Cloud Credentials entry view.  
```

File: content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md [hunk 3]
```diff
@@ -29,7 +29,21 @@ Go to **Data Protection > Cloud Sync Tasks** and click **Add**.
...
+{{< expand "What happens if my cloud sync credentials are invalid?" "v" >}}
+Once you chose a cloud credential from the dropdown list, TrueNAS automatically validates access to that cloud sync provider.  Invalid credentials will result in the following Alert: 
+
+![DataProtectionCloudSyncInvalidAlert](/images/SCALE/DataProtectionCloudSyncInvalidAlert.png "Invalid Credentials Alert")
+
+Click **FIX CREDENTIAL**.  You will be directed to the Cloud Credentials entry view.  
+
+![DataProtectionCloudSyncInvalidFix](/images/SCALE/DataProtectionCloudSyncInvalidFix.png "Name and Provider View")
+
+Check your provider credentials and update the applicable fields within the ***Authentication*** section, click ***Verify Credential***.  *The Credential is valid* will be displayed if TrueNAS successfully accessed your provider.  Click ***Save*** and return to **Data Protection > Cloud Sync Tasks > Add**.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/Cloud Sync Tasks/_index.md:43]:
```suggestion
Check your provider credentials and update the applicable fields within the ***Authentication*** section, then click ***Verify Credential***.  *The Credential is valid* will display if TrueNAS successfully accesses your provider.  Click ***Save*** and return to **Data Protection > Cloud Sync Tasks > Add**.
```

**Comment by Mrt134** [comment]
symantec.com is still down - approved for merge.

**Pull request merged**